mars/StepDefinitions/LanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,121 @@
+using Mars.Pages;
+using Mars.Support;
+using NUnit.Framework;
+
+namespace Mars.StepDefinitions
+{
+    [Binding]
+    public class LanguagesStepDefinitions: BaseTest
+    {
+
+        LoginPage loginPage;
+        HomePage homePage;
+        LanguagePage languagePage;
+        ProfilePage profilePage;
+        int row_count =0;
+
+        public LanguagesStepDefinitions(FeatureContext featureContext) : base(featureContext)
+        {
+
+        }
+        [Given(@"navigate to the language tab")]
+        public void GivenNavigateToTheLanguageTab()
+        {
+            languagePage = new LanguagePage();
```

Comment:
i would initialise all instance variables in a constructor please make use of this 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

Reply:
Totally Agree, I have improved my test scripts according your comments. Thanks